### ISSUE TITLE
Why is the reverse logic in MenuItem.php isActive () at line 89 ?

### ISSUE THREAD

**Issue by jerabina** [contributte/menu-control#42]
```
if (!$this->include) {
	$actionName = sprintf('%s:%s', $presenter->getName(), $presenter->getAction());
	foreach ($this->include as $include) {
		if (preg_match(sprintf('~%s~', $include), $actionName)) {
			return $this->active = true;
		}
	}
}
```

**Comment by foxycode** [comment]
@jerabina Looks like a bug. Can you create PR with test?

**Comment by foxycode** [comment]
@f3l1x I need your help.

I fixed `include` logic in commit https://github.com/contributte/menu-control/commit/1bae3eada5eb47131303bcc53809d57bce5b262c
When you have lowest depenencies: `composer update --prefer-dist --prefer-lowest`
`$presenter->getName()` for some reason returns `NULL` and I don't know why.

**Comment by f3l1x** [comment]
I'll put on my todo-list.

**Comment by foxycode** [comment]
Looks like a Nette bug, fixed in commit https://github.com/contributte/menu-control/commit/f606e8f42407066aaf7877f367fcd5f42c9e36ed (closes #89)

@jerabina Sorry it took so long.
